frontend/views/containers/forms/FormLogin.vue

```diff
@@ -8,24 +8,22 @@ form(
 )
   label.field
     i18n.label Username
-    input.input#loginName(
+    input.input(
       :class='{error: $v.form.name.$error}'
       name='name'
-      @keyup.enter='login'
-      @input='debounceName'
-      ref='username'
-      autofocus
+      v-model='form.name'
+      @input='debounceField("name")'
+      @blur='updateField("name")'
       data-test='loginName'
-      v-error:name='{ tag: "p", attrs: { "data-test": "badUsername" } }'
+      v-error:name='{ attrs: { "data-test": "badUsername" } }'
+      autofocus
     )
 
-  // TODO #661 - improve :v
   form-password(
     :label='L("Password")'
-    :value='form'
-    :v='$v.form'
-    @enter='login'
-    @input='(newPassword) => {password = newPassword}'
+    :$v='$v'
+    @input='debounceField("password")'
+    @blur='updateField("password")'
```

Comment:
Are you positive you need these? Can you check it see if it's possible to get it working without them?

I noticed that in FormPassword.vue you are now binding the `v-model` to the `.$model`, which might make this no longer necessary?

If you try and are unable to get rid of them, can you write a comment here and/or in FormPassword.vue explaining why it these are required?

Reply:
checking...

Reply:
Yep, that's right, I added them directly to FormPassword and it works!